src/Aws/Common/Credentials/Credentials.php

```diff
@@ -147,7 +147,7 @@ public static function fromIni($profile = null, $filename = null)
             throw new \RuntimeException("Invalid AWS credentials file: {$filename}.");
```

Comment:
I think this block of code needs to be updated to not thrown when the credentials file is empty. `parse_ini_file()` could return `[]` on an empty file, which would cause an exception. Perhaps instead we should check if the result if `=== false`.